docs/model_zoo.md

```diff
@@ -129,7 +129,7 @@ Please refer to [CGNet](https://github.com/open-mmlab/mmsegmentation/blob/master
 
 ### Mixed Precision (FP16) Training
 
-Please refer [Mixed Precision (FP16) Training](https://github.com/open-mmlab/mmsegmentation/blob/master/configs/fp16) for details.
+Please refer [Mixed Precision (FP16) Training] on BiSeNetV2 (https://github.com/open-mmlab/mmsegmentation/blob/master/configs/bisenetv2/bisenetv2_fcn_fp16_4x4_1024x1024_160k_cityscapes.py) for details.
```

Comment:
It seems that some algorithms are missing in the model zoo, should add in the future.